internal/query/permission.go

```diff
@@ -35,3 +36,17 @@ func wherePermittedOrgs(ctx context.Context, query sq.SelectBuilder, filterOrgId
 		filterOrgIds,
 	)
 }
+
+func wherePermittedOrgsOrCurrentUser(ctx context.Context, query sq.SelectBuilder, filterOrgIds, orgIDColumn, userIdColum, permission string) sq.SelectBuilder {
+	userID := authz.GetCtxData(ctx).UserID
+	fmt.Printf("userID = %+v\n", userID)
+	logging.WithFields("permission_check_v2_flag", authz.GetFeatures(ctx).PermissionCheckV2, "org_id_column", orgIDColumn, "user_id_colum", userIdColum, "permission", permission, "user_id", userID).Debug("permitted orgs check used")
+
+	return query.Where(
+		fmt.Sprintf(wherePermittedOrgsOrCurrentUserClause, orgIDColumn, userIdColum, userID),
+		authz.GetInstance(ctx).InstanceID(),
+		userID,
+		permission,
+		filterOrgIds,
+	)
```

Comment:
Pass the userID as argument


```suggestion
	return query.Where(
		fmt.Sprintf(wherePermittedOrgsOrCurrentUserClause, orgIDColumn, userIdColum, userID),
		authz.GetInstance(ctx).InstanceID(),
		userID,
		permission,
		filterOrgIds,
        userID,
	)
```